ipaserver/install/bindinstance.py

```diff
@@ -327,9 +327,9 @@ def read_reverse_zone(default, ip_address, allow_zone_overlap=False):
         if not allow_zone_overlap:
             try:
                 dnsutil.check_zone_overlap(zone, raise_on_error=False)
-            except ValueError as e:
+            except dnsutil.DNSZoneAlreadyExists as e:
                 logger.error("Reverse zone %s will not be used: %s",
-                             zone, e)
+                             zone, e.msg)
```

Comment:
``.msg`` is deprecated. ``DNSException`` sets ``self.msg = str(self)`` if keyword args are passed. ``e`` is good enough since the ``%s`` will call ``str(e)`` any way.